src/components/organisms/AssetContent/MetaFull.tsx

```diff
@@ -3,11 +3,25 @@ import MetaItem from './MetaItem'
 import styles from './MetaFull.module.css'
 import Publisher from '../../atoms/Publisher'
 import { useAsset } from '../../../providers/Asset'
+import {
+  IVerifiablePresentation,
+  ServiceMetadataMarket
+} from '../../../@types/MetaData'
 
 export default function MetaFull(): ReactElement {
-  const { ddo, metadata, isInPurgatory, type } = useAsset()
-  const { algorithm } = ddo.findServiceByType('metadata').attributes.main
-
+  const { ddo, type, isServiceSelfDescriptionVerified } = useAsset()
```

Comment:
extract this and use the code created in the prev comment

Reply:
Fixed